primitives/core/src/assertion.rs

```diff
@@ -23,16 +23,18 @@ use sp_runtime::{traits::ConstU32, BoundedVec};
 
 type MaxStringLength = ConstU32<64>;
 pub type ParameterString = BoundedVec<u8, MaxStringLength>;
-pub type IndexingNetworks = BoundedVec<IndexingNetwork, MaxStringLength>;
+pub type IndexingNetworks = BoundedVec<SupportedNetwork, MaxStringLength>;
```

Comment:
Is this intentionally kept as `IndexingNetworks` ?

Reply:
Yes